blog/authors.yml

```diff
@@ -1,10 +1,16 @@
-jaimergp:
-  name: Jaime Rodríguez-Guerra
-  title: Steering council member
-  url: https://github.com/jaimergp
-  image_url: https://github.com/jaimergp.png
+# Alphhabetical by author ID please.
```

Comment:
```suggestion
# Alphabetical by author ID please.
```

Reply:
OK, you have to admit that misspelling alphabetical is kind of funny.